frontend/src/scenes/CallingPage.tsx

```diff
@@ -102,6 +103,21 @@ export default function CallingPage({ locale, routePath }: SceneProps) {
   const { me: user } = userState || {};
   const [contactState, contactService] = useContactService();
   const { activeContact } = contactState || {};
+  const [isFeedbackDialogOpen, setIsFeedbackDialogOpen] = useState(false);
+
+  const exitCallingPage = () => {
+    userService?.setShouldSleep(true);
```

Comment:
This should probably be moved into `handleExitCallButton()`. We want to prevent sleeping only when a call is active.

Reply:
Gotcha!

Reply:
Sorry, could you draw the link a little more explicitly for me? I don't see how the code you attached ties in with moving around the `setShouldSleep()` call.

That said, upon reading this again, I realise that your implementation preserves the existing behaviour (we only reenable sleeping when the user exits the page, not when the call is disconnected). I don't believe in asking you to fix current behaviour while you're adding a new feature, so feel free to ignore the code comments :)

Reply:
Okies! Haha... Oh I think I was under the impression that if you set user to sleep, user might become `null` but then realized it's a misunderstanding on my end. Think can try moving it to `handleExitCall()` to make it cleaner.